addons/html_builder/static/src/core/building_blocks/builder_fontfamilypicker.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
 <t t-name="html_builder.BuilderFontFamilyPicker">
-    <BuilderSelect>
+    <BuilderSelect id="'fontFamilyPicker'">
```

Comment:
This will creates multiple instances of this button with the same ID, which is not good 😬 
<img width="291" height="657" alt="Capture d’écran 2025-09-30 à 13 21 52" src="https://github.com/user-attachments/assets/c7edec0a-b130-450c-8a79-7e063b4f588a" />

What about simply add `lh-1` in `<BuilderSelectItem> > div` in `line 8` ?